tests/integ/test_multithreading.py

```diff
@@ -664,6 +664,89 @@ def change_config_value(session_):
     )
 
 
+@pytest.mark.xfail(
```

Comment:
How does this test ensures that large query breakdown works well with multithreading if it's marked as xfail?

Reply:
It is only disabled for local-testing since large query optimization does not work with local testing.